sim/druid/forms.go

```diff
@@ -263,144 +263,126 @@ func (druid *Druid) registerCatFormSpell() {
-// 				druid.AutoAttacks.SetReplaceMHSwing(druid.ReplaceBearMHFunc)
-// 				druid.AutoAttacks.EnableAutoSwing(sim)
-
-// 				druid.manageCooldownsEnabled()
-// 				druid.UpdateManaRegenRates()
-// 			}
-// 		},
-// 		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
-// 			druid.form = Humanoid
-// 			druid.AutoAttacks.SetMH(druid.WeaponFromMainHand())
-
-// 			druid.PseudoStats.ThreatMultiplier /= 2.1021
-// 			druid.PseudoStats.SchoolDamageDealtMultiplier[stats.SchoolIndexPhysical] /= 1.0 + 0.02*float64(druid.Talents.MasterShapeshifter)
-// 			druid.PseudoStats.DamageTakenMultiplier /= potpdtm
-//			Switch to using AddStat as PseudoStat is being removed
-// 			druid.PseudoStats.BaseDodge -= 0.02 * float64(druid.Talents.FeralSwiftness+druid.Talents.NaturalReaction)
-
-// 			druid.AddStatsDynamic(sim, predBonus.Invert())
-// 			druid.AddStatsDynamic(sim, statBonus.Invert())
-// 			druid.RemoveDynamicEquipScaling(sim, stats.Armor, druid.BearArmorMultiplier())
-// 			if potpDep != nil {
-// 				druid.DisableDynamicStatDep(sim, potpDep)
-// 			}
-
-// 			healthFrac := druid.CurrentHealth() / druid.MaxHealth()
-// 			druid.DisableDynamicStatDep(sim, stamDep)
-// 			if hotwDep != nil {
-// 				druid.DisableDynamicStatDep(sim, hotwDep)
-// 			}
-// 			druid.RemoveHealth(sim, druid.CurrentHealth()-healthFrac*druid.MaxHealth())
-
-// 			if !druid.Env.MeasuringStats {
-// 				druid.AutoAttacks.SetReplaceMHSwing(nil)
-// 				druid.AutoAttacks.EnableAutoSwing(sim)
-
-// 				druid.manageCooldownsEnabled()
-// 				druid.UpdateManaRegenRates()
-// 				druid.EnrageAura.Deactivate(sim)
-// 				druid.MaulQueueAura.Deactivate(sim)
-// 			}
-// 		},
-// 	})
-
-// 	rageMetrics := druid.NewRageMetrics(actionID)
-
-// 	furorProcChance := []float64{0, 0.2, 0.4, 0.6, 0.8, 1}[druid.Talents.Furor]
-
-// 	druid.BearForm = druid.RegisterSpell(Any, core.SpellConfig{
-// 		ActionID: actionID,
-// 		Flags:    core.SpellFlagNoOnCastComplete | core.SpellFlagAPL,
-
-// 		ManaCost: core.ManaCostOptions{
-// 			BaseCost:   0.55,
-// 			Multiplier: (1 - 0.2*float64(druid.Talents.KingOfTheJungle)) * (1 - 0.1*float64(druid.Talents.NaturalShapeshifter)),
-// 		},
-// 		Cast: core.CastConfig{
-// 			DefaultCast: core.Cast{
-// 				GCD: core.GCDDefault,
-// 			},
-// 			IgnoreHaste: true,
-// 		},
-
-// 		ApplyEffects: func(sim *core.Simulation, _ *core.Unit, spell *core.Spell) {
-// 			rageDelta := 0 - druid.CurrentRage()
-// 			if sim.Proc(furorProcChance, "Furor") {
-// 				rageDelta += 10
-// 			}
-// 			if rageDelta > 0 {
-// 				druid.AddRage(sim, rageDelta, rageMetrics)
-// 			} else if rageDelta < 0 {
-// 				druid.SpendRage(sim, -rageDelta, rageMetrics)
-// 			}
-// 			druid.BearFormAura.Activate(sim)
-// 		},
-// 	})
-// }
+func (druid *Druid) registerBearFormSpell() {
+	actionID := core.ActionID{SpellID: core.TernaryInt32(druid.Level < 40, 5487, 9634)}
+
+	statBonus := druid.GetFormShiftStats().Add(stats.Stats{
+		stats.AttackPower: 3 * float64(druid.Level),
```

Comment:
Trivial, but I recently discovered that Form Stats doesn't use druid.level in game, it actually uses Floor(FeralCombatSkill / 5). So when using the feral combat ring rune, you end up getting +3 AP and + 32 health, or something like that. Same for cat.

We can probably skip this though if it's not a trivial change.